external-crates/move/crates/move-vm-runtime/src/execution/mod.rs

```diff
@@ -6,3 +6,4 @@ pub mod interpreter;
 pub mod tracing;
 pub mod values;
 pub mod vm;
+pub use crate::jit::execution::ast::Type;
```

Comment:
Sensible, I suppose -- where is this used?

Reply:
This is used in parts of the adapter since we need to look at the `Type`s in the `LoadedFunctionInformation` to determine whether arguments to a function are references or not. 

If we add something like `TypeTagWithRefs`, we could probably remove this exposure.